components/user/UserTabs.js

```diff
@@ -12,13 +12,14 @@ export default function UserTabs({ tabs, setTabs }) {
       )
     );
   };
-
+  
   return (
     <div>
       <div className="sm:hidden">
         <label htmlFor="tabs" className="sr-only">
           Select a tab
         </label>
+        {tabs.length>1 && // if tabs contains more than one options, then show dropdown menu
```

Comment:
I think the code is self explanatory, but I think the formatting is not the correct coding style, VScode should correct this, you might need to install the Prettier plugin